src/NHibernate/Multi/LinqBatchItem.cs

```diff
@@ -7,46 +7,45 @@
 using NHibernate.Util;
 using Remotion.Linq.Parsing.ExpressionVisitors;
 
-namespace NHibernate
+namespace NHibernate.Multi
 {
 	public partial class LinqBatchItem<T> : QueryBatchItem<T>
 	{
-		private static Delegate _postExecuteTransformer;
-		private static NhLinqExpression _linqEx;
```

Comment:
Fixed. Static state for things not supposed to be common between instances is a bit evil!

Reply:
yeah.. That's some messed auto refactoring thing that I didn't notice.